_inc/client/plans/single-product-backup.jsx

```diff
@@ -12,32 +12,30 @@ import { withRouter } from 'react-router';
  */
 import analytics from 'lib/analytics';
 import Button from 'components/button';
-import ExternalLink from 'components/external-link';
 import PlanPrice from 'components/plans/plan-price';
 
 import './single-product-backup.scss';
 
-export function SingleProductBackup( { plan, products, upgradeLinks, isFetchingData } ) {
-	// Don't show the product card for paid plans.
-	if ( ! isFetchingData && 'jetpack_free' !== plan ) {
-		return null;
-	}
+export function SingleProductBackup( props ) {
+	const { isFetching, ...backupCardProps } = props;
 
 	return (
-		<React.Fragment>
-			<div className="plans-section__single-product">
-				{ isFetchingData ? (
-					<div className="plans-section__single-product-skeleton is-placeholder" />
-				) : (
-					<SingleProductBackupCard products={ products } upgradeLinks={ upgradeLinks } />
-				) }
-			</div>
-		</React.Fragment>
+		<div className="plans-section__single-product">
+			{ isFetching ? (
+				<div className="plans-section__single-product-skeleton is-placeholder" />
+			) : (
+				<SingleProductBackupCard { ...backupCardProps } />
```

Comment:
I like it better when we know what the exact props are that we are passing though. instead of using the spread operator. I think it make it much clearer in terms of what props are being used.

Reply:
Totally agree with Enej here.

Reply:
This is addressed in e8f995a.